src/utils.ts

```diff
@@ -1,3 +1,20 @@
 export function normalizeDiacritics(text: string): string {
   return text.normalize('NFD').replace(/\p{Diacritic}/gu, '');
 }
+
+export function calculateFontSize(
+  doc: PDFKit.PDFDocument,
+  text: string,
+  minFontSize: number,
+  maxFontSize: number,
+  maxWidth: number
```

Comment:
Please consider an object for this function to make the function call a bite more self explanatory.
```
// now
calculateFontSize(doc, `»${capstoneProject.title}«`, 10, 28, 200)

// then
calculateFontSize({
  doc, 
  text: `»${capstoneProject.title}«`, 
  minFontSize: 10, 
  maxFontSize: 28,
  maxWidth: 200
})
```